### ISSUE TITLE
Simple auto-indent feature use

### ISSUE THREAD

**Issue by MrYann** [James-Yu/LaTeX-Workshop#201]
Hi : )

As you surely know, VSCode just implemented an auto-indent feature. I think that it could be good for latex documents, since I like to indent all the begin--end blocks. So, here is a very basic suggestion to append to your "syntax.json" file: 
```json
	"indentationRules": {
		"increaseIndentPattern": "(\\\\begin{(?!document))|(\\\\documentclass)",
		"decreaseIndentPattern": "(\\\\end{)|(\\\\begin{document})"
	}
```

Note that unfortunately VSCode does not seem to correct indentation within pasted blocks, so you have to write your own lines to test the feature (after setting `editor.autoIndent` to true)

**Comment by James-Yu** [comment]
Actually I did not study the release note as I am on travel :p I noticed you merged recent commits to your fork. Would you like to have a PR soon on this?

**Comment by MrYann** [comment]
Nevermind my merge: I tried to update my fork with your current code, to be able to propose the modification as a pull request. However, when I try a new pull request, it says they are hundreds of changes in what I try tu pull to you. That is why I simply opened this issue to show you what could be done, since it is a very small modification.

**Comment by James-Yu** [comment]
Merged in #204 . Thanks for the contribution!
